backend/danswer/direct_qa/answer_question.py

```diff
@@ -0,0 +1,82 @@
+import time
```

Comment:
just to be sure, you moved out the 2 OpenAI pieces but no functional changes to these files right?

Reply:
Correct, just moved somewhere else so it can be re-used for the slack bot + adding error message to the response